syntax = "proto3";

package sbai_tak_bridge_protos;

message StateUpdateRequest {
  CortexState requested_state = 1;
}

enum CortexState {
  STATE_UNSPECIFIED = 0;
  INIT = 1;
  HIBERNATE = 2;
  STANDBY = 3;
  TELEOP = 4;
  SOLO_AUTO = 5;
  SWARM_AUTO = 6;
}